src/kOS/Suffixed/AggregateResourceValue.cs

```diff
@@ -1,44 +1,41 @@
 ﻿using System.Collections.Generic;
 using kOS.Safe.Encapsulation;
+using kOS.Safe.Encapsulation.Suffixes;
+using kOS.Suffixed.Part;
 
 namespace kOS.Suffixed
 {
     public class AggregateResourceValue : Structure
     {
         private readonly string name;
+        private readonly SharedObjects shared;
         private double amount;
         private double capacity;
-        private readonly ListValue parts;
+        private readonly SuffixedList<PartValue> parts;
```

Comment:
The reason, like i mentioned above, was to make one that we could use for everything and would let us enforce at compiletime that it was a return value that would be useful for scripters. because that failed I have to admit it is not cool enough to exist and i will remove it.